src/shell/commands/duplication.cpp

```diff
@@ -135,12 +135,13 @@ void attach_dups_stat(const list_dups_stat &stat, dsn::utils::multi_table_printe
     printer.add_row_name_and_data("unfinished_app_count", stat.unfinished_app_count);
 
     // Add stats for duplications.
-    printer.add_row_name_and_data("duplication_count", stat.duplication_count);
+    printer.add_row_name_and_data("total_duplication_count", stat.duplication_count);
     for (const auto &[status, cnt] : stat.dup_status_stats) {
-        printer.add_row_name_and_data(fmt::format("{}_count", status), cnt);
+        printer.add_row_name_and_data(fmt::format("duplication_count_by_status({})", status), cnt);
     }
     for (const auto &[remote_cluster, cnt] : stat.dup_remote_cluster_stats) {
-        printer.add_row_name_and_data(fmt::format("{}_count", remote_cluster), cnt);
+        printer.add_row_name_and_data(
+            fmt::format("duplication_count_by_follower_cluster({})", remote_cluster), cnt);
```

Comment:
Are the row names are too long, the displayed table row may be too wide and hard to be filled on screen ?

Could you please give a sampe output?

Reply:
OK, it's key/value pair under a summary tag:

```
[summary]
total_app_count              : 8
duplicating_app_count        : 3
unfinished_app_count         : 0
duplication_count            : 3
DS_LOG_count                 : 3
target_cluster_count         : 3
total_partition_count        : 64
duplicating_partition_count  : 24
unfinished_partition_count   : 0
```